### PR TITLE
[FIX: #67] Added some animations.

### LINKED ISSUE #67: animate.css is Included but not used

### PR DESCRIPTION
**Pull request by Xizt**
Let me know if some more/less animations required.

### CODE REVIEW

**Comment by Xizt** [comment]
Animations added are : index.html -> HEADING , dates of dev sprint.
Remaining pages just the HEADINGS are animated.
Link for this PR: https://xizt.github.io/inpycon2017/index.html

File: index.html [hunk 1]
```diff
@@ -24,6 +24,8 @@
 	<link rel="stylesheet" href="css/colors/color-3.css">
 	<link rel="stylesheet" href="css/responsive.css">
 	<script src="js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
+	<script src="js/wow.min.js"></script>
+    <script>new WOW().init();</script>
```
Comment (CuriousLearner) [inline index.html]: Move this to custom js file.

**Review by CuriousLearner — COMMENTED** [review]
Hey, 

Thanks for your contributions!

I'm not sure if this is really needed.

However, I've mentioned some review. I will wait for others to look here. If they give a green flag, then only I'll merge this.

@realslimshanky Like to look at this?
@theskumar What do you think about this?

**Comment by realslimshanky** [comment]
Animations looks great. I would like to recommend few enhancements.
* Add the [license](https://raw.githubusercontent.com/graingert/WOW/master/LICENSE) file for this library as well on the repo.
* On the homepage, add 'fadeInDown' animation to 'Speakers', 'Sponsors' and 'Latest Tweets' also.
* If you know bootstrap as well shift the 'Speakers' heading to the centre on the homepage since all other headings are. If you cannot just let me know I'll create and issue and work on it.
Thanks for your contribution :+1:

**Comment by Xizt** [comment]
I'll push an updated PR with all the changes by tonight :)

**Comment by Xizt** [comment]
Updated the changes. Let me know if there are more of em.

**Review by realslimshanky — APPROVED** [review]
LGTM

**Pull request merged**